tests/framework/validators/NumberValidatorTest.php

```diff
@@ -37,7 +69,13 @@ public function testValidateValueSimple()
         $this->assertTrue($val->validate(-20));
         $this->assertTrue($val->validate('20'));
         $this->assertTrue($val->validate(25.45));
+
+        $this->setPointDecimalLocale();
         $this->assertFalse($val->validate('25,45'));
+        $this->setCommaDecimalLocale();
+        $this->assertTrue($val->validate('25,45'));
```

Comment:
did this pass before? what if the value if passed to a database which does not understand `,` as decimal separator?